README.md

```diff
@@ -62,7 +62,7 @@ Sync native files
 npx cap sync
 ```
 
-On iOS, Android and Web, no further steps are needed.
+On iOS, Android and Web, no further steps are needed by stricly following the files location then just use the filename as *assetPath*.
```

Comment:
This is better but the whole line is untrue if you need to open the project in xcode to get it working